src/twister2/plugin.py

```diff
@@ -95,6 +100,21 @@ def pytest_addoption(parser: pytest.Parser):
         metavar='PATH',
         help='load hardware map from a file',
     )
+    twister_group.addoption(
+        '-G', '--integration',
```

Comment:
only added to comamnds, not handled yet

Reply:
You can extend function `validate_options` for checking if `--integration` was selected and inform users that this option is not implemented yet.

Reply:
I will move `--integration` to the next PR with implementation of platform scope. But good to know and remember `validate_options`, to avoid using dummy combination of options.